packages/perseus/src/widgets/radio/__tests__/multiple-choice.test.ts

```diff
@@ -0,0 +1,893 @@
+import {describe, beforeEach, it} from "@jest/globals";
```

Comment:
I figured it made sense to have a completely separate test file, as it's likely that we may have different test conditions with the upcoming new designs. Plus, it's easier to remember that we need to clean it up at the end of the project. ;)